README.md

```diff
@@ -88,4 +93,77 @@ Assuming you know the `<zookeeper_address>` and have placed your config file nam
 
 `docker run -it -v <config_dir_on_host>:/etc/kafka-configurator skyuk/kafka-configurator -f=/etc/kafka-configurator/test-topics.yml --zookeeper=<zookeeper_address>`
 
-Alternatively you could extend the `skyuk/kafka-configurator` image and `COPY` your configuration file directly into your extended image.
\ No newline at end of file
+Alternatively you could extend the `skyuk/kafka-configurator` image and `COPY` your configuration file directly into your extended image.
+
+
+
+## Usage for adding as a dependency
+
+##### SBT
+```sbtshell
+useJCenter := true
+
+libraryDependencies += "com.sky" %% "kafka-configurator" % "VERSION"
+
+```
+
+##### Gradle
+```groovy
+repositories {
+    jcenter()
+}
+
+compile 'com.sky:kafka-configurator_2.12:VERSION'
+```
+
+##### Maven
+```xml
+<repositories>
+    <repository>
+        <id>jcenter</id>
+        <name>jcenter</name>
+        <url>http://jcenter.bintray.com</url>
+    </repository>
+</repositories>
+
+<dependency>
+  <groupId>com.sky</groupId>
+  <artifactId>kafka-configurator_2.12</artifactId>
+  <version>VERSION</version>
+</dependency>
+```
+
+### Scala Example:
+```
```

Comment:
Specify Scala syntax please: https://help.github.com/articles/creating-and-highlighting-code-blocks/